build/gulpfile.vscode.js

```diff
@@ -478,9 +478,9 @@ function packageTask(platform, arch, sourceFolderName, destinationFolderName, op
 				// --- Start Positron ---
 				// These are commented out since they're not currently used in 'resources/win32/bin/code.sh'
 				// .pipe(replace('@@VERSION@@', version))
-				// .pipe(replace('@@POSITRONVERSION@@', positronVersion))
-				// .pipe(replace('@@BUILDNUMBER@@', positronBuildNumber))
 				// --- End Positron ---
+				.pipe(replace('@@POSITRONVERSION@@', positronVersion))
```

Comment:
Just need to move this within the Positron code fence since it's additional code that isn't from upstream.

Reply:
I've made the change in the other branch and CI is running